LICENSE

```diff
@@ -1,202 +1,202 @@
+Apache License
```

Comment:
This change confuses GitHub's license detection mechanism ([licensee](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)). Better don't touch this file and keep using the old one.